foundation-extensions/experience-platform-extension/README.md

```diff
@@ -16,22 +16,26 @@ The Adobe Experience Platform Edge Network extension requires the Identity for E
 4. Click **Save**.
 5. Follow the publishing process to update SDK configuration.
 
-### Configure the Edge Network extension
-
 ![Edge Network extension configuration](../../.gitbook/assets/mobile-edge-launch-configuration.png)
```

Comment:
can we update this screenshot to not show the install Identity info box since this is a prerequisite (see Install Identity for Edge Network above) and also show a datastream selected?